testing-modules/junit-5-advanced/src/test/java/com/baeldung/getTestName/JUnit4ParameterizedTestNameUnitTest.java

```diff
@@ -0,0 +1,43 @@
+package com.baeldung.getTestName;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TestName;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(Parameterized.class)
+public class JUnit4ParameterizedTestNameUnitTest {
+
+    private final String input;
+    private final String expected;
```

Comment:
Let's add a new line after these fields. There's also no strong reason to make them `final`. It's not _wrong_ but it doesn't add much.